app/routes/_index.tsx

```diff
@@ -23,6 +27,20 @@ export const meta: V2_MetaFunction = () => {
 
 export default function Index() {
   const tekster = useTekster(ESanitySteg.FORSIDE);
+  const [samtykkeErBekreftet, settSamtykkeErBekreftet] = useState(false);
+  const [feilmeldingAktivert, settFeilmeldingAktivert] = useState(false);
```

Comment:
Tror dette fikses på i en annen PR men nevner det her også:
Jeg og Halvor hadde en idè om at vi starter med en konvensjon om at navnet på boolean states alltid skal starte med `er` bare for å være konsekvente på det.
```suggestion
  const [erSamtykkeBekreftet, settErSamtykkeBekreftet] = useState(false);
  const [erFeilmeldingAktivert, settErFeilmeldingAktivert] = useState(false);
```